resin/exceptions.py

```diff
@@ -313,3 +313,63 @@ def __init__(self, required_version, current_version):
         self.code = 'UnsupportedFunction'
         self.exit_code = 1
         self.message = Message.SUPERVISOR_VERSION_ERROR.format(req_version=required_version, cur_version=current_version)
+
+
+class AmbiguousApplication(Exception):
+    """
+    Exception of Resin ambiguous application.
+
+    Args:
+        application (str): application name.
+
+    Attributes:
+        code (str): exception code.
+        exit_code (int): program exit code.
+        message (str): error message.
+
+    """
+
+    def __init__(self, application):
+        self.code = 'AmbiguousApplication'
+        self.exit_code = 1
```

Comment:
I meant I've not had time to clearly define other exit codes for every case so for now, only 1 as exit code for all errors